deploy/docker/Dockerfile

```diff
@@ -1,6 +1,8 @@
-FROM BASEIMAGE
+FROM scratch
 
 COPY heapster eventer /
 COPY ca-certificates.crt /etc/ssl/certs/
 
+#   nobody:nobody
+USER 65534:65534
```

Comment:
Note: if you put a username:group here instead, it uses the mapping in /etc/passwd, but since this is from scratch, there is no mapping there. The alternative would be to make our own, but that seems unnecessary.